docs/source/cloud-setup/cloud-permissions.rst

```diff
@@ -1,6 +1,6 @@
 .. _cloud-permissions:
 
-Minimal Cloud Permissions
+Cloud Account Permissions
```

Comment:
Any reasoning behind this change?

Reply:
Ahh, this was a remnant. Reverted. Thanks!